plugins/candela/web_client/views/ConfigView.js

```diff
@@ -0,0 +1,26 @@
+import PluginConfigBreadcrumbWidget from 'girder/views/widgets/PluginConfigBreadcrumbWidget';
+import View from 'girder/views/View';
+
+import ConfigViewTemplate from '../templates/configView.pug';
+
+var ConfigView = View.extend({
+    initialize: function (settings) {
+        this.render();
+    },
+
+    render: function () {
+        this.$el.html(ConfigViewTemplate());
+
+        if (!this.breadcrumb) {
+            this.breadcrumb = new PluginConfigBreadcrumbWidget({
+                pluginName: 'Candela file visualizer',
+                el: this.$('.g-config-breadcrumb-container'),
+                parentView: this
+            }).render();
```

Comment:
This would break if `render` were called twice.  I would move the construction of `this.breadcrumb` to `initialize` and change this to
```
this.breadcrumb.setElement(this.$('.g-config-breadcrumb-container')).render()
```